backend/models/group.js

```diff
@@ -335,46 +316,30 @@ Group.create = async function (account, model, branch = "master", rid = null, se
 };
 
 Group.deleteGroups = async function (account, model, sessionId, ids) {
-	const groupIds = [].concat(ids);
-
-	ids = [].concat(ids).map(x => utils.stringToUUID(x));
+	const groupIds = ids.map(utils.stringToUUID);
+	await db.remove(account, getGroupCollectionName(model), { _id: { $in: groupIds } });
 
-	const deleteResponse = await db.remove(account, getGroupCollectionName(model), { _id: { $in: ids } });
-
-	if (!deleteResponse.result.ok) {
-		throw responseCodes.GROUP_NOT_FOUND;
-	}
-
-	ChatEvent.groupsDeleted(sessionId, account, model, groupIds);
+	ChatEvent.groupsDeleted(sessionId, account, model, ids);
 };
 
 Group.deleteGroupsByViewId = async function (account, model, view_id) {
 	return await db.remove(account, getGroupCollectionName(model), { view_id });
 };
 
-Group.findByUID = async function (account, model, branch, revId, uid, showIfcGuids = false, noClean = true) {
+Group.findByUID = async function (account, model, branch, revId, uid, showIfcGuids = false, noClean = true, convertToIfcGuids = false) {
```

Comment:
what's the difference between showIfcGuids and convertToIfcGuids?

Reply:
@carmenfan They're flags for different things. It was introduced when I merged `findByUID` with another function that was there before. `convertToIfcGuids` decides if it goes through the `getObjectsArrayAsIfcGuids` or `getObjectIds` function, and `showIfcGuids` is used within the functions that `getObjectIds` calls.

Easiest solution would probably be renaming it (and possibly renaming all the other functions because they all sound similar).

Another possibility is rewriting the functions that set `group.objects` so that it makes more sense.